service/src/main/kotlin/io/provenance/digitalcurrency/consortium/service/BankService.kt

```diff
@@ -120,9 +113,6 @@ class BankService(
                         check(registration.isActive()) { "Cannot transfer to removed bank account $bankAccountUuid" }
                         CoinTransferRecord.insert(uuid, registration, amount)
                     }
-                    // Sending to another member bank
-                    blockchainAddress != null && pbcService.getMembers().members.any { it.id == blockchainAddress } ->
```

Comment:
we're still validating the blockchain address but we aren't actually using it to do anything with this bit removed.

Reply:
sorry not sure i follow, any reason to keep this?

Reply:
if we're going to allow a transfer request to a block chain address then yes. If not then we should take the blockchain address off the `TransferRequest` object.